### PR TITLE
Update registrymgr.h by adding more functions definitions

### PR DESCRIPTION
**Pull request by CelesteBlue-dev**

### CODE REVIEW

File: include/psp2/registrymgr.h [hunk 1]
```diff
@@ -82,6 +82,99 @@ int sceRegMgrSetKeyInt(const char* category, const char* name, int buf);
...
+/**
+ * Get a system param key's information by id
+ *
+ * @param id - The id of the key
+ * @param buf - Pointer to a char buffer to hold the value
+ * @param size - The size of the buffer
+ *
+ * @return 0 on success, < 0 on error
+ */
+int sceRegMgrSystemParamGetInt(const int id, int* buf);
```
Comment (d3m3vilurr) [inline include/psp2/registrymgr.h]: wrong params :P

File: include/psp2/registrymgr.h [hunk 2]
```diff
@@ -82,6 +82,99 @@ int sceRegMgrSetKeyInt(const char* category, const char* name, int buf);
...
+/**
+ * Get all keys' initial information by category (from os0:kd/registry.db0)
+ *
+ * @param category - The path to the directory to be opened (e.g. /CONFIG/SYSTEM)
+ * @param buf - Pointer to a char buffer to hold the values
+ * @param elements_number - The number of elements of the buffer
+ *
+ * @return 0 on success, < 0 on error
+ */
+int sceRegMgrGetInitVals(const char* category, char* buffer, const int elements_number);
```
Comment (d3m3vilurr) [inline include/psp2/registrymgr.h]: please match document's param name and function argument

File: include/psp2/registrymgr.h (hunk 1 above)
Comment (CelesteBlue-dev) [reply include/psp2/registrymgr.h]: ;-) thanks

File: include/psp2/registrymgr.h (hunk 2 above)
Comment (CelesteBlue-dev) [reply include/psp2/registrymgr.h]: yeah

File: include/psp2/registrymgr.h [hunk 3]
```diff
@@ -82,6 +82,99 @@ int sceRegMgrSetKeyInt(const char* category, const char* name, int buf);
  */
 int sceRegMgrSetKeyStr(const char* category, const char* name, char* buf, const int size);
 
+/**
+ * Get all keys' initial information by category (from os0:kd/registry.db0)
+ *
+ * @param category - The path to the directory to be opened (e.g. /CONFIG/SYSTEM)
+ * @param buf - Pointer to a char buffer to hold the values
```
Comment (devnoname120) [inline include/psp2/registrymgr.h]: `[out]`

**Comment by devnoname120** [comment]
@CelesteBlue-dev Could you update your pull request?

**Pull request closed without merging**